lib/rules/no-unused-emit-declarations.js

```diff
@@ -56,6 +56,17 @@ function hasReferenceId(value, setupContext) {
   )
 }
 
+/**
+ * Check if the given name matches emitReferenceIds variable name
+ * @param {string} name
+ * @param {Set<Identifier>} emitReferenceIds
+ * @returns {boolean}
+ */
+function verifyEmitVariableName(name, emitReferenceIds) {
```

Comment:
I think we should rename this function:

```suggestion
function isEmitVariableName(name, emitReferenceIds) {
```

"verify" sounds like there is a "correct" and a "wrong" way.